lib/permission-api.js

```diff
@@ -189,64 +78,60 @@ module.exports = {
       return Promise.reject(new Error(`'${accessLevel}' is not a valid access level. Must be ${offerAccessLevels.join(', ')}.`));
     }
 
-    const mayRead = true;
-    const mayWrite = accessLevel === 'write' || accessLevel === 'admin';
-    const mayManage = accessLevel === 'admin';
-
-    const permissionOffer = {
-      id: generateUniqueId(),
-      createdAt: new Date(),
-      updatedAt: new Date(),
-      expiresAt,
-      realmUrl,
-      mayRead,
-      mayWrite,
-      mayManage
+    const options = {
+      method: 'POST',
+      headers: { Authorization: this.token },
+      body: {
+        expiresAt,
```

Comment:
This looks wrong? Shouldn't this be a `"expiresAt": expiresAt`?

Reply:
They are equivalent.